examples/display_camera/sdkconfig.bsp.esp32_s3_korvo_2

```diff
@@ -2,11 +2,15 @@
 # Espressif IoT Development Framework (ESP-IDF) Project Minimal Configuration
 #
 CONFIG_IDF_TARGET="esp32s3"
+CONFIG_ESPTOOLPY_FLASHMODE_QIO=y
 CONFIG_ESPTOOLPY_FLASHSIZE_4MB=y
-CONFIG_ESP_DEFAULT_CPU_FREQ_240=y
```

Comment:
Why the CPU frequency cannot be at maximum?

Reply:
I'll restore it.